<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>641</height>
     </rect>
    </property>
    <widget class="QWidget" name="mainMenuPage">
     <widget class="QLabel" name="mainTitle">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Asymetric Cipher and Key Exchange</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="mainMenuGroup">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>231</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="exitBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="elgamalPageBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>ElGamal</string>
       </property>
      </widget>
      <widget class="QPushButton" name="dhPageBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Diffie-Hellman</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rsaPageBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>RSA</string>
       </property>
      </widget>
     </widget>
     <zorder>mainMenuGroup</zorder>
     <zorder>mainTitle</zorder>
    </widget>
    <widget class="QWidget" name="rsaPage">
     <widget class="QLabel" name="rsaTitle">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>RSA</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="elgamalPage">
     <widget class="QLabel" name="elgamalTitle">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ElGamal</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dhPage">
     <widget class="QLabel" name="dhTitle">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Diffie-Hellman</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
